WordPress/src/main/java/org/wordpress/android/ui/stats/StatsAbstractListFragment.java

```diff
@@ -131,6 +132,19 @@ public void onCreate(Bundle savedInstanceState) {
     @Override
     public void onSaveInstanceState(Bundle outState) {
         //AppLog.d(AppLog.T.STATS, this.getTag() + " > saving instance state");
+
+        // Do not serialize VolleyError, but rewrite in a simple stats Exception.
+        // VolleyErrors should be serializable, but for some reason they are not.
+        // FIX for https://github.com/wordpress-mobile/WordPress-Android/issues/2228
+        if (mDatamodels != null) {
+            for (int i=0; i < mDatamodels.length; i++) {
+                if (mDatamodels[i] != null && mDatamodels[i] instanceof VolleyError) {
+                    VolleyError currentVolleyError = (VolleyError) mDatamodels[i];
+                    mDatamodels[i] = StatsUtils.rewriteVolleyError(currentVolleyError, getString(R.string.error_refresh_stats));
```

Comment:
If an error error object is received from the StatsService the UI displays the error message to the user (for some kind of errors, not all).
When the error is received, it's stored in the mDatamodel[i], and restored upon resume, like any other datatypes.

If mDatamodel[i] is null, an empty UI is shown, that says something like "No stats data for this period".
So to reply to your question, yes we can use null, in fact in the first version of this patch I had set the datamodel to null, but then added the logic to save the error and keep the UI is consistent upon device rotation or resume.
Otherwise, if we set it to null and the user rotates the device the UI is rebuilt with an empty mDatamodel and the "No stats data for this period" is shown.